events/app/views/components/event-promo/event-promo.njk

```diff
@@ -0,0 +1,15 @@
+{% set sizes = data.sizes if data.sizes else '100vw' %}
```

Comment:
Where is this from? I can't see where data is passed in?

Reply:
Almost guaranteed that we'll need it (copied over from the current `promo` template), but you're right – I should take it out until such time as we're using it.